src/cli/cli_syntax.md

```diff
@@ -1,22 +1,143 @@
-## CLI Syntax
+# CLI Syntax
+Before you start reading this chapter I highly recommend you the read the [POSIX Utility Conventions][POSIX Utility Conventions]
```

Comment:
Delete the crossed-out `the`: "recommend you ~~the~~ read the POSIX Utility Conventions"

Reply:
Also, end this line with a period: "[POSIX Utility Conventions]**.**"